doc/source/Tutorials/Scripting/CreatingConfigurationFiles3/demos/demoMacbethChart.gfr

```diff
@@ -0,0 +1,154 @@
+__children["Outputs"]["outputs"]["output1"]["parameters"]["displayHost"].addChild( Gaffer.StringPlug( "value", defaultValue = 'localhost', flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Outputs"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["InteractiveAppleseedRender"] = GafferAppleseed.InteractiveAppleseedRender( "InteractiveAppleseedRender" )
+parent.addChild( __children["InteractiveAppleseedRender"] )
+__children["InteractiveAppleseedRender"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Catalogue"] = GafferImage.Catalogue( "Catalogue" )
+parent.addChild( __children["Catalogue"] )
+__children["Catalogue"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["latlong_map_environment_edf"] = GafferAppleseed.AppleseedLight( "latlong_map_environment_edf" )
+parent.addChild( __children["latlong_map_environment_edf"] )
+__children["latlong_map_environment_edf"].loadShader( "latlong_map_environment_edf" )
+__children["latlong_map_environment_edf"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Group"] = GafferScene.Group( "Group" )
+parent.addChild( __children["Group"] )
+__children["Group"]["in"].addChild( GafferScene.ScenePlug( "in1", flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Group"]["in"].addChild( GafferScene.ScenePlug( "in2", flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Group"]["in"].addChild( GafferScene.ScenePlug( "in3", flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Group"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Camera"] = GafferScene.Camera( "Camera" )
+parent.addChild( __children["Camera"] )
+__children["Camera"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["as_disney_material"] = GafferOSL.OSLShader( "as_disney_material" )
+parent.addChild( __children["as_disney_material"] )
+__children["as_disney_material"].loadShader( "material/as_disney_material" )
+__children["as_disney_material"].addChild( Gaffer.V2fPlug( "__uiPosition", defaultValue = imath.V2f( 0, 0 ), flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+__children["Reference"] = Gaffer.Reference( "Reference" )
+parent.addChild( __children["Reference"] )
+__children["Reference"].load( "/home/michaeldu/dev/gafferDependencies-0.53.0.0-linux/resources/references/macbethTexture.grf" )
```

Comment:
This breaks the documentation build, because it references a file that doesn't exist in the repo. It seems there might be another issue with the original PR this came from too - Tom is going to comment on this at source. It seems it is time we added the documentation build to Travis CI...

Reply:
Is it worth backing out https://github.com/GafferHQ/gaffer/pull/2943 until we can fix the build/broken references issues?

Reply:
Oh I think that Macbeth chart was PRed to the resources repo

Reply:
Not that that solves the absolute filepath problem

Reply:
Found it, thanks https://github.com/GafferHQ/resources/pull/4

Reply:
@andrewkaufman @johnhaddon @medubelko I'm fixing some of this up as part of https://github.com/GafferHQ/gaffer/pull/3108, but that won't be ready till next week as were off here tomorrow/Monday.

I'm including the `.grf`/`.png` with the distribution resources, so the Tutorial that talks about custom node registration will be ok, but it won't be able to include a sample `.gfr` file with a render setup until we fix the path issues.

Reply:
Let's make this as easy as possible, and skip #2943 for this backport.

Just for record-keeping: we were aware of the issue with the missing reference. I had closed https://github.com/GafferHQ/resources/pull/4, thinking that we would have .grf copying in the build sorted before the next backport.

Reply:
Dropped